### DISCUSSION TITLE
Server title and players count does not displayed

### DISCUSSION THREAD

**Discussion by 3bdulrahmanOthman** [aXenDeveloper/ips-app-advanced-serverlist#42] [Q&A]
Hello, I tried to use your app in the forum but I found this problem, can you help me fix it?
![412](https://user-images.githubusercontent.com/86932829/162859209-71197921-24b1-4cbd-902b-915786754473.png)

**Comment by KuroRaiken** [comment]
@Supremache You may have blocked ports on your hosting.

**Comment by Rushaway** [comment]
Wich ports need to be opened ? We need more details on this please. Docs is realy unclear on this point.
